ocaml/tests/test_vm_migrate.ml

```diff
@@ -12,57 +12,62 @@
  * GNU Lesser General Public License for more details.
  *)
 
-open OUnit
-open Test_common
-
 let mac1 = "00:00:00:00:00:01"
 let mac2 = "00:00:00:00:00:02"
 
+let check_network_map =
+  Alcotest.(check (list
+                  (pair
+                    (Alcotest_comparators.ref ())
+                    (Alcotest_comparators.ref ()))))
```

Comment:
Since we're not using `slist`, this will break when the two lists have the same elements but in different order